tests/frameworks/gtest/include/gtest/internal/gtest-param-util.h

```diff
@@ -598,6 +599,16 @@ class ParameterizedTestCaseInfo : public ParameterizedTestCaseInfoBase {
     }  // for test_it
   }  // RegisterTests
 
+  std::vector<std::pair<std::string, TestMetaFactoryBase<typename TestCase::ParamType>*>> GetTestMetaFactories() {
+    std::vector<std::pair<std::string, TestMetaFactoryBase<typename TestCase::ParamType>*>> toReturn;
+    for (typename TestInfoContainer::iterator test_it = tests_.begin(); test_it != tests_.end(); ++test_it) {
```

Comment:
+1, also auto would be nice

---
In reply to: [98142566](https://github.com/Microsoft/WinObjC/pull/1849#discussion_r98142566) [](ancestors = 98142566)